includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -130,6 +139,24 @@ public static function render_vary_header_field() {
 		<?php
 	}
 
+	/**
+	 * Render content negotiation field.
+	 */
+	public static function render_content_negotiation_field() {
+		$value = \get_option( 'activitypub_content_negotiation', '1' );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" id="activitypub_content_negotiation" name="activitypub_content_negotiation" value="1" <?php checked( '1', $value ); ?> />
+				<?php \esc_html_e( 'Differentiates how content is shown to browsers versus Fediverse services.', 'activitypub' ); ?>
```

Comment:
```suggestion
				<?php \esc_html_e( 'Enable content negotiation for browsers and Fediverse services.', 'activitypub' ); ?>
```
A bit more explicit about what the setting does and makes it clear that enabling/disabling is the action.